cplus.py

```
@@ -0,0 +1,7 @@
+for (int i = 0; i < n; i += 4) {
```

Ruff (0.8.2)

SyntaxError: Expected ')', found name

---

SyntaxError: Expected ':', found '='

---

SyntaxError: Expected a statement

---

SyntaxError: Expected an expression

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected '}', found 'for'

---

SyntaxError: Expected 'in', found newline

---

Statement ends with an unnecessary semicolon

(E703)